Wikipedia/Code/MWKSection.m

```diff
@@ -280,7 +280,7 @@ - (NSString*)summary {
 - (nullable NSArray<MWKTitle*>*)disambiguationTitles {
     NSArray* textNodes = [self elementsInTextMatchingXPath:WMFSectionDisambiguationTitlesXPathSelector];
     return [textNodes wmf_mapAndRejectNil:^id (TFHppleElement* node) {
-        if (node.text.length == 0 || [node.text containsString:@"redlink=1"]) {
+        if (node.text.length == 0 || [node.text containsString:@"redlink=1"] || [node.text containsString:@"cite_note"]) {
```

Comment:
also, we should really have some tests for this. even simple "expected output for this sample string" tests